configure.ac

```diff
@@ -187,6 +187,7 @@ PKG_CHECK_MODULES([EOS_KNOWLEDGE_PRIVATE], [
     epoxy
 ])
 PKG_CHECK_MODULES([GRESOURCE_PLUGIN], [
+    eknr-0,
```

Comment:
Oh, that was all it needed? Excellent

Reply:
In theory, that's all that's needed. I wasn't able to end-to-end test this as specified on the phabricator comment, but this will get the resource loaded into the process' memory space and there should not be any shadowing issues.